README.md

```diff
@@ -7,14 +7,21 @@ For pre-commit: see https://github.com/pre-commit/pre-commit
 
 For clang-format: see https://github.com/ssciwr/clang-format-wheel
 
+In comparison to other available pre-commit hooks for clang-format, this one installs clang-format from Python wheels.
+This makes it usable without additional installation requirements - e.g. in pre-commit CI.
 
 ### Using clang-format with pre-commit
 
-Add this to your `.pre-commit-config.yaml`:
+Add this to your `.pre-commit-config.yaml` for a project containing C, C++, C# or CUDA files:
```

Comment:
I want to keep this text the same as what it is in the other repositories -- perhaps below the blob you could write what the default `types_or` and `args` are and link to the docs about changing them rather than prose